corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -128,7 +128,6 @@ def test_bad_integer_datatype(self):
         self.assertEqual(len(bad_ints), self.adapter.get_query_object().count())
 
 
-@temporarily_enable_toggle(KAFKA_UCRS, 'user-reports')
```

Comment:
i was testing this by running the whole `userreports` suite (since the toggle was causing a later test to fail). i was curious whether the decorator would work here and didn't notice that it was just causing the test not to run (which also resulted in the later test passing).